<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="cvwidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>331</width>
      <height>471</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Basic</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>99</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Image directory</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>27</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>410</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Defaults</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtering:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="medianCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Median Filter</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="medianSizeSpin">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>57</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="gaussCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gaussian Filter</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="gaussSizeSpin">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>57</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="thresholdCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Thresholding</string>
      </property>
     </widget>
     <widget class="QSlider" name="thresholdSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSlider" name="valueSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="thresholdLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="valueLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>valueLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="greenRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="blueRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="redRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Advanced</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Frame Averaging:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="frameSpin">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>57</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Frames</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="fileDirectory">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>811</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>placeholder for directory</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture image</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
